### PR TITLE
fix the nfs repo and addon test

### PR DESCRIPTION
**Pull request by dashea**
Or at least fix the anaconda side of it, the repo itself is still pretty busted.

### CODE REVIEW

File: tests/kickstart_tests/make-addon-pkgs.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/python3
+
+# This script creates all the packages used by nfs-repo-and-addon.ks.
+# The packages are created in two directories, http and nfs. After all the rpms
+# are made just copy everything to the locations set in $KSTEST_ADDON_HTTP_REPO
+# and $KSTEST_ADDON_NFS_REPO.
+#
+# This script imports things from tests/lib/mkdud.py, so tests/lib needs to be
+# in $PYTHONPATH.
+
```
Comment (bcl) [inline tests/kickstart_tests/make-addon-pkgs.py]: May as well add the standard copyright statement here.

**Comment by bcl** [comment]
I like the use of rpmfluff here!

**Comment by vpodzime** [comment]
Looks good to me too otherwise.

**Comment by dashea** [comment]
Corrected and pushed.

**Pull request closed without merging**